### ISSUE TITLE
Buttercup 2.0.0. not working in linux

### ISSUE THREAD

**Issue by BlackGopher2030** [buttercup/buttercup-desktop#995]
When i download buttercup and execute it: `./Buttercup-linux-x64-2.0.0.AppImage`

I get this error:

```
[INF] 09:28:59: Application ready
[INF] 09:28:59: Application session started: 2021-04-12T09:28:59.394Z
[INF] 09:28:59: Logs location: /home/zeus/.local/state/Buttercup-nodejs/buttercup-desktop.log
[INF] 09:28:59: System locale detected: en-US
[INF] 09:28:59: Starting with language: en-US
[INF] 09:28:59: Updated search records for 0 vaults
[INF] 09:28:59: Updated search records for 0 vaults
[INF] 09:28:59: Updated search records for 0 vaults
[ERR] 09:28:59: TypeError: electron_1.systemPreferences.canPromptTouchID is not a function
at <anonymous>              …KvP/resources/app.asar/build/main/services/biometrics.js:82  return electron_1.systemPreferences.canPromptTouchID();                         
at next                                                                                                                                                                   
at                          …JKvP/resources/app.asar/build/main/services/biometrics.js:8  step((generator = generator.apply(thisArg, _arguments || [])).next());          
at new Promise                                                                                                                                                            
at __awaiter                …JKvP/resources/app.asar/build/main/services/biometrics.js:4  return new (P || (P = Promise))(function (resolve, reject) {                    
at supportsBiometricUnlock  …KvP/resources/app.asar/build/main/services/biometrics.js:81  return __awaiter(this, void 0, void 0, function* () {                           
at                          …r7zJKvP/resources/app.asar/build/main/actions/appMenu.js:34  const biometricsSupported = yield biometrics_1.supportsBiometricUnlock();       
at next                                                                                                                                                                   
at fulfilled                …er7zJKvP/resources/app.asar/build/main/actions/appMenu.js:5  function fulfilled(value) { try { step(generator.next(value)); } catch (e) { re…

```

This happened after the update of the fingerprint (last update)

**Comment by Alternus** [comment]
This bug has already been reported in #993 & #990, it's an issue where the `systemPreferences.canPromptTouchID` function doesn't exist in environments other then MacOS. I've created a PR which resolves the issue (Tested it on Manjaro) #994 just waiting for it to get approved.

**Comment by perry-mitchell** [comment]
Fixed by #994, sorry for the inconvenience.
